Tests/GRPCCodeGenTests/Internal/Translator/SnippetBasedTranslatorTests.swift

```diff
@@ -487,7 +487,7 @@ extension SnippetBasedTranslatorTests {
     try XCTAssertEqualWithDiff(contents, expectedSwift)
   }
 
-  private func assertThrowsError<T, E: Error>(
+  internal func assertThrowsError<T, E: Error>(
```

Comment:
This should be moved to the test utilities file, it's worth renaming it to `XCTAssertThrowsError` too